lib/hw_unique_key/Kconfig

```diff
@@ -63,3 +63,14 @@ config HW_UNIQUE_KEY_PARTITION_SIZE
 	help
 	  Size of the flash page for storing the HUK.
 	  Only needed when there is no KMU for storing keys (nRF52840).
+
+config HW_UNIQUE_KEY_READ
+	bool "Use HUK"
+	depends on !BUILD_WITH_TFM
+	select SECURE_BOOT if HAS_HW_NRF_ACL
+	select FPROTECT if HAS_HW_NRF_ACL
+	help
+	  Use the Hardware Unique Key (HUK) in read access to derive new keys.
+	  It will require an immutable bootloader with the CONFIG_HW_UNIQUE_KEY_LOAD
+	  option set on devices without a KMU.
+	  Note that the key has to be written before.
```

Comment:
Btw, the key has to be written before what?

Reply:
before it can be read. 
Is that enough or do you think that requires more details

Reply:
I would maybe add more details. Is that something that user has to make sure that it's done? And how. Maybe reference to or mention of a function/option/procedure that is used, if any.

Reply:
Oh was not aware that we include these procedure description in Kconfigs,

I added a reference to the Kconfigs used for that, I will describing the whole process here is a bit too much

Reply:
Yeah, no need to describe the entire process.
It just needed some more info, because writing just simply that the key has to be written before it is read is kind of obvious and does not give that much of a useful info.
But writing that the key has to be written by XX before it can be read, or XX has to have completed and written the key before it can be read gives a clear instruction on what users should be looking out for.
Giving these references is useful I think, so if you are happy now, then I am as well :)

Reply:
Yeah makes sense, but I think the reference to the Kconfig that is needed for writing should be fine now